mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -6255,10 +6256,82 @@ struct ThreadwiseCopyRewritePattern
         else
           destLowerIndices = destUpperIndices;
 
+        bool toEmitOOBStoreCheckLogic = false;
+        SmallVector<unsigned, 2> oobStoreCheckDims;
+        if (destLowerIndices.size() == 5) {
+          for (unsigned iter = 0; iter < destTransform.getNumResults();
```

Comment:
please give an example to explain logic here.

Reply:
the logic of toEmitOOBStoreCheck remove now ,
toEmitOOBStoreCheckLogic = false;
will discuss with letao how to enable it